packages/components/scripts/vendorism/custom/textarea.vendorism.js

```diff
@@ -20,11 +20,6 @@ const transformComponent = (path, originalContent) => {
     ['filled', 'readonly'],
     // Make sure we don't unobserve an undefined element
     // @todo: Remove when shoelace ships this fix
```

Comment:
I would remove this also, so we don't wonder in the future, why this was there and if filtering for `todo`'s it won't appear

```suggestion
```